projects/wallet-template/README.md

```diff
@@ -1,2 +1,105 @@
-## Substrate Connect Wallet Template
+# Substrate Connect - Light Client Wallet Template
 
+## Overview
+
+Welcome to Substrate Connect - the ultimate light client wallet template! Substrate
+Connect leverages the power of the smoldot,
```

Comment:
should it be `the power of smoldot` or is referring to smoldot as `the smoldot` more correct?

Reply:
power of smoldot is fine